specs/commandline/blob.md

```diff
@@ -70,24 +70,29 @@ Flags:
 ### notation blob sign
 
 ```text
-Produce a signature for a given blob. A detached signature file will be written to the currently working directory with blob file name + ".sig" + signature format as the file extension. For example, signature file name for "myBlob.bin" will be "myBlob.bin.sig.jws" for JWS signature format or "myBlob.bin.sig.cose" for COSE signature format.
+Produce a detached signature for a given blob.
+
+The signature file will be written to the currently working directory with file name "{blob file name}.{signature format}.sig".
 
 Usage:
-  notation blob sign [flags] <blob_path>
+  notation blob sign [flags] <blobPath>
```

Comment:
Which naming convention are we using? `snake_case` or `camelCase`? We need to make it consistent.